src/FunderMaps.Core/Types/FoundationDamageCause.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace FunderMaps.Core.Types
+namespace FunderMaps.Core.Types
```

Comment:
Again? Whats this?

Reply:
Files on origin were encoded in UTF-8 with BOM. This changes the file to UTF-8 without BOM.